examples/cpp/pyperf/PyPerfType.h

```diff
@@ -213,6 +216,10 @@ typedef struct event {
   // hashmap with Symbols and only store the ids here
   int32_t stack_len;
   int32_t stack[STACK_MAX_LEN];
+  uint64_t user_ip;
+  uint64_t user_sp;
+  uint32_t user_stack_len;
+  uint8_t raw_user_stack[]; // NOTICE: Field with variadic length - must be last!
 #define FRAME_CODE_IS_NULL ((int32_t)0x80000001)
```

Comment:
Keep this define near `int32_t stack[STACK_MAX_LEN];` as it's a special value for that array.